jenkins/Jenkinsfile.linux

```diff
@@ -351,9 +307,9 @@ pipeline {
 
                 environment {
                   PACKAGE_DIR = "package/linux/${BUILD_LOCATION}"
-                  PACKAGE_FILE = "${renameFile(PACKAGE_DIR, 'rstudio-*.' + ext_map[env.OS])}"
+                  PACKAGE_FILE = "${utils.renameFile(PACKAGE_DIR, 'rstudio-*.' + ext_map[env.OS])}"
                   TAR_PACKAGE_DIR= "${PACKAGE_DIR}/_CPack_Packages/Linux/${ext_map[env.OS].toUpperCase()}"
-                  TAR_PACKAGE_FILE = "${renameTarFile(TAR_PACKAGE_DIR)}"
+                  TAR_PACKAGE_FILE = "${utils.renameTarFile(TAR_PACKAGE_DIR)}"
```

Comment:
```suggestion
                  TAR_PACKAGE_FILE = "${(FLAVOR == "Desktop" || FLAVOR == "Electron") ? utils.renameTarFile(TAR_PACKAGE_DIR) : ''}"
```

Something like this, where we use a ternary here instead of putting it inside the utils function, just to make it clear that we want to perform an action in this case or leave it blank otherwise.

Reply:
I'm _assuming_ that you can do a ternary here. I'm pretty sure you can in ordinary Groovy strings, but I'm not sure about the environment directive.